README

```diff
@@ -0,0 +1,37 @@
+# AMSE/SAKI 2023 Template Project
+This template project provides some structure for your open data project in the AMSE/SAKI module.
+
+Before you begin, make sure you have [Python](https://www.python.org/) and [Jayvee](https://github.com/jvalue/jayvee) installed. We will work with [Jupyter notebooks](https://jupyter.org/). The easiest way to do so is to set up [VSCode](https://code.visualstudio.com/) with the [Jupyter extension](https://marketplace.visualstudio.com/items?itemName=ms-toolsai.jupyter).
+
+## Project Setup
+The following files are part of this template repository as examples and should be **replaced by you** over the semester:
+
+- `data.sqlite`: Your final, cleaned dataset. You will create an automated data pipeline that creates this SQLite database from multiple open data sources. The template repository includes data about train stations in germany, you need to replace this with your data!
+- `exploration.ipynb`: A Jupyter notebook that you can use to explore your data and show in detail what it looks like. You can refer to this file in your report for users that want more information about your data.
+- `report.ipynb`: Your final report as a Jupyter notebook. This is the result of your project work and should lead with a question that you want to answer using open data. The content of the report should answer the question, ideally using fitting visualizations, based on the data in `data.sqlite`.
+
+
+## Exercises
+During the semester you will need to complete exercises, sometimes using [Python](https://www.python.org/), sometimes using [Jayvee](https://github.com/jvalue/jayvee). You **must** place your submission in the `exercises` folder in your repository and name them according to their number from one to five: `exercise<number from 1-5>.<jv or py>`. At the end of the semester, you must have the following files in your repository for full marks:
```

Comment:
Last sentence potentially leaves a wrong impression that there is a final submission at the end of the semester. Also would note that they will be informed separately if they should do jv or python in the corresponding week.